packages/paste-core/components/display-pill-group/__tests__/index.spec.tsx

```diff
@@ -0,0 +1,126 @@
+import * as React from 'react';
+import {render} from '@testing-library/react';
+import {matchers} from 'jest-emotion';
+import {CustomizationProvider} from '@twilio-paste/customization';
+// @ts-ignore typescript doesn't like js imports
+import axe from '../../../../../.jest/axe-helper';
+import {DisplayPillGroup, DisplayPill} from '../src';
+import {Basic, CustomDisplayPillGroup} from '../stories/index.stories';
+
+expect.extend(matchers);
+
+describe('DisplayPillGroup', () => {
+  // Verifies that the correct aria attributes and semantics are met
+  describe('Rendered shape', () => {
+    it('should render correctly', () => {
+      const {getByTestId, getByText} = render(<Basic />);
+      expect(getByText('Tennis')).toBeDefined();
+
+      const group = getByTestId('display-pill-group');
+      expect(group.getAttribute('aria-label')).toBe('Your favorite sports:');
+      expect(group.tagName).toBe('UL');
+
+      const pillLink = getByTestId('display-pill-anchor');
+      expect(pillLink.getAttribute('href')).toBe('https://google.com');
+      expect(pillLink.getAttribute('rel')).toBe('noreferrer noopener');
+      expect(pillLink.getAttribute('target')).toBe('_blank');
+      expect(pillLink.tagName).toBe('A');
+
+      const pillBlank = getByTestId('display-pill-blank');
```

Comment:
Nit: terming this as "Blank" at first made me think this pill would have nothing inside of it, which is not the case. Would a more clear way to name this be "plain" or "default"?